### PR TITLE
Buffs the fuck out of space drugs.

### LINKED ISSUE #15063: Space drugs are lame
I mean, their low-dosage effects haven't changed a ton, but high doses used to be a lot more interesting. At 999U, you'd hallucinate things attacking you constantly, hear a barrage of voices, and fall over every 5 seconds. Now there's no difference between 50U and 900.

After testing, I think hallucination itself may have been nerfed. Even manually setting hallucination to 999 doesn't have the same effect as it used to.

### PR DESCRIPTION
**Pull request by Buggy123**
"ok to be honest all their effects are lame as fuck" - freerealestate

They make you hallucinate way fucking harder now.

fixes #15063

### CODE REVIEW

**Comment by Niknakflak** [comment]
>listening to demas for valid opinions

**Comment by Iamgoofball** [comment]
>doxxing freerealestate

you monster

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
 
 /datum/reagent/drug/space_drugs/overdose_process(mob/living/M)
-	if(M.hallucination < volume && prob(20))
-		M.hallucination += 5
+	if(M.hallucination < volume*10)
```
Comment (neersighted) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: spaces please

**Comment by phil235** [comment]
I'd like to point out that I kept hallucinations from space drug overdose pretty weak by design. I didn't want people complaining that I ruined their favorite drug.

**Comment by Buggy123** [comment]
It's space drugs, you're supposed to hallucinate. Before ODing was added it made you hallucinate.

**Comment by freerealestate** [comment]
Niknak my dick is getting sore from you constantly riding it in OOC, the forums and GitHub 

Also i appreciate the intention behind this PR, making space drugs actually work like drugs, but what I meant was that their effects have always been lame, i.e. the hallucinations are annoying and gay

**Comment by PKPenguin321** [comment]
uhh buggy that's not true at all
space drugs before ODing made you see colors, mindbreaker made you hallucinate. space drugs never made you hallucinate until goofchem

**Comment by KorPhaeron** [comment]
wait yeah why are you making space drugs identical to mindbreaker we already have mindbreaker

**Comment by Dorsisdwarf** [comment]
:-1: space drugs are for trippy drooling, mindbreaker hallucinates (it's LSD)

**Comment by PKPenguin321** [comment]
Yeah, don't merge this. Needs to be closed.

**Comment by Cheridan** [comment]
>wait yeah why are you making space drugs identical to mindbreaker we already have mindbreaker

**Comment by Buggy123** [comment]
Oh shit i totally thought mindbreaker and space drugs were the same thing.

**Comment by PKPenguin321** [comment]
based cheri

**Comment by freerealestate** [comment]
cheery
make the hallucinations not gay

**Pull request closed without merging**